### ISSUE TITLE
[Omoda] Changed version 5.6 to 4.4?

### ISSUE THREAD

**Issue by deku-m** [alexbelgium/hassio-addons#457]
**Which addon?**
Omoda

name: Feature request
about: Suggest an idea for this project
title: ''
labels: ''
assignees: ''

---

**Which addon?**

**Addon or app?**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
The last update changed the docker container version from 5.6 to 4.4 is this by accident?

**Comment by alexbelgium** [comment]
Omada is linked to "latest", so normally it should still be 5.6. The automatic version numbering script must have messed up somehow

**Comment by alexbelgium** [comment]
Should be good now. However, latest is 5.5, not 5.6

**Comment by alexbelgium** [comment]
I'll close it as code changed

**Comment by Deckoz2302** [comment]
Looks like it is updates for alternative architectures. Chromium, vs chromium amd64, vs arm64. Docker images are probably updated at different times for each platform. I think Alex's bots just pull the latest for each platform. Which may not actually change the image on your hardware, but creates a committed update in the repo due to upstream version change.

**Comment by alexbelgium** [comment]
Exactly! and the code fetched the most recent version out of the 10 last pushed containers. However, they had recently pushed legacy versions, which caused the numbering to fall back to 4.4. I've updated my updated code so that it sorts through the last 100 pushed containers. This allows going back to 5.5 :-)
